src/main/java/pl/touk/sputnik/review/Review.java

```diff
@@ -36,11 +37,11 @@
     private List<String> messages = new ArrayList<>();
     private Map<String, Short> scores = new HashMap<>();
 
-    private ReviewFormatter formatter;
+    private final ReviewFormatter formatter;
 
-    public Review(@NotNull List<ReviewFile> files) {
+    public Review(@NotNull List<ReviewFile> files, Configuration configuration) {
```

Comment:
It would be easier to construct ReviewFormatter outside and pass it to constructor.